### PR TITLE
Conn 1926

### PR DESCRIPTION
**Pull request by jsmith2**
Some cert management clean up and adding refresh capabilities.

### CODE REVIEW

**Comment by jsmith2** [comment]
Do not merge before outstanding Common-Types pr is merged.

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/impl/CertificateManagerServiceImpl.java
Comment (pvenkatakrishnan1) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/impl/CertificateManagerServiceImpl.java:126]: Instead of UnsupportedOperationException, can throw CertificateManagerException with a not supported yet operation msg. (optional)

**Review by pvenkatakrishnan1 — APPROVED** [review]
Looks good.

File: Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/CertificateManagerImpl.java
Comment (trantang) [inline Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/CertificateManagerImpl.java:244]: should used MessageFormat.format - make code easier to read

**Review by trantang — APPROVED** [review]
Did not see any problem

**Pull request merged**
